lib/es6/processHelpers.js

```diff
@@ -5,12 +5,12 @@ let spawn = require("child_process").spawn;
 let exec = require("child_process").exec;
 
 let processHelpers = {
-    run: function (command) {
+    run: function (command, silent) {
```

Comment:
There should be no explicit options arguments on utility helpers. Please use an options object, which have a silent property with default value of "inherit" (by using lodash's _.defaults).